lib/tasks/scheduler.rake

```diff
@@ -11,8 +11,6 @@ task house_valuation_collector: :environment do
       Sentry.capture_exception(e, extra: { house_id: house.id })
     end
   end
-
-  HappyHood::Slack::Client.send_daily_price_summary
```

Comment:
I'll add a rake task in Heroku when this is merged in that will run at 9 AM and sends an update.

The rake task that gathers pricing information will be pushed back to 8:30 AM to give us plenty of time to be ready for the daily update.